src/hotspot/share/classfile/stackMapTable.cpp

```diff
@@ -210,45 +225,56 @@ VerificationType StackMapReader::parse_verification_type(u1* flags, TRAPS) {
   return VerificationType::bogus_type();
 }
 
-StackMapFrame* StackMapReader::next(
-    StackMapFrame* pre_frame, bool first, u2 max_locals, u2 max_stack, TRAPS) {
+StackMapFrame* StackMapReader::next(TRAPS) {
+  _parsed_frame_count++;
+  check_size(CHECK_NULL);
+  StackMapFrame* frame = next_helper(CHECK_VERIFY_(_verifier, nullptr));
+  if (frame != nullptr) {
+    check_offset(frame);
```

Comment:
`check_offset` can install a verifier error, so I think you need to check for that!